drivers/SmartThings/matter-appliance/fingerprints.yml

```diff
@@ -37,6 +37,12 @@ matterGeneric:
     deviceTypes:
       - id: 0x007A
     deviceProfileName: extractor-hood
+  - id: "matter/extractor-hood/on-off/light"
+    deviceLabel: Matter Extractor Hood/OnOff Light
+    deviceTypes:
+      - id: 0x007A
+      - id: 0x0100
+    deviceProfileName: extractor-hood-light-binary
```

Comment:
What's the meaning of `binary` in this context?

Reply:
The OnOff light device type in matter-switch fingerprints to the `light-binary` profile, so I borrowed that same name here. Would just `light` be better?